From 84c287e49568a8c8512c9a8602985020dabf313f Mon Sep 17 00:00:00 2001
From: Cosimo Cecchi <cosimoc@gnome.org>
Date: Fri, 04 Feb 2011 18:44:53 +0000
Subject: css: buttons can be active without be hovered

For instance, GtkToggleButton implementations which are not radio/check
buttons; those were themed as normal buttons, which makes them not
recognizable as active.
The gradient for this is stolen from a previous version of this same CSS
file.
---
diff --git a/themes/Adwaita/gtk-3.0/gtk.css b/themes/Adwaita/gtk-3.0/gtk.css
index 2deb742..f70368b 100644
--- a/themes/Adwaita/gtk-3.0/gtk.css
+++ b/themes/Adwaita/gtk-3.0/gtk.css
@@ -263,6 +263,13 @@ GtkScrollbar.button:prelight:active {
 				     to (#e5e7e5));
 }
 
+.button:active {
+    background-image: -gtk-gradient (linear,
+    				     left top, left bottom,
+				     from (shade (@bg_color, 0.86)),
+				     to (shade (@bg_color, 1.09)));
+}
+
 .button:hover:active,
 .toolbar .button:hover:active {
     background-image: -gtk-gradient (linear,
--
cgit v0.8.3.1
